diff --git a/dist/runtime-dom.d.ts b/dist/runtime-dom.d.ts
index 3366f5ad1b2aa9bdd34435f5a6d4cc2c20f8241f..81242e042269b9948e9c0bbf42ea5f21a05f8688 100644
--- a/dist/runtime-dom.d.ts
+++ b/dist/runtime-dom.d.ts
@@ -1491,24 +1491,6 @@ type NativeElements = {
   >
 }
 
-declare global {
-  namespace JSX {
-    interface Element extends VNode {}
-    interface ElementClass {
-      $props: {}
-    }
-    interface ElementAttributesProperty {
-      $props: {}
-    }
-    interface IntrinsicElements extends NativeElements {
-      // allow arbitrary elements
-      // @ts-ignore suppress ts:2374 = Duplicate string index signature.
-      [name: string]: any
-    }
-    interface IntrinsicAttributes extends ReservedProps {}
-  }
-}
-
 // suppress ts:2669
 export {}
 