diff --git a/types/jsx.d.ts b/types/jsx.d.ts
index 85c9c1df23d949bbce4748f2350ec1a1c9a354ea..5b7ae4eecda0f5da115d8af32653d22d2f343c8c 100644
--- a/types/jsx.d.ts
+++ b/types/jsx.d.ts
@@ -1326,26 +1326,5 @@ import {
   AllowedComponentProps
 } from './vnode'
 
-declare global {
-  namespace JSX {
-    interface Element extends VNode {}
-    interface ElementClass {
-      $props: {}
-    }
-    interface ElementAttributesProperty {
-      $props: {}
-    }
-    interface IntrinsicElements extends NativeElements {
-      // allow arbitrary elements
-      // @ts-ignore suppress ts:2374 = Duplicate string index signature.
-      [name: string]: any
-    }
-    interface IntrinsicAttributes
-      extends ReservedProps,
-        AllowedComponentProps,
-        ComponentCustomProps {}
-  }
-}
-
 // suppress ts:2669
 export {}